lib/Doctrine/ORM/Query/SqlWalker.php

```diff
@@ -751,7 +751,9 @@ public function walkSelectClause($selectClause)
             foreach ($class->associationMappings as $assoc) {
                 if ( ! ($assoc['isOwningSide'] && $assoc['type'] & ClassMetadata::TO_ONE)) {
                     continue;
-                } else if ( !$addMetaColumns && !isset($assoc['id'])) {
+                }
+                
+                if ( !$addMetaColumns && !isset($assoc['id'])) {
```

Comment:
@guiajlopes according to our CS we need spaces before AND after the negation operator, so this should be:

```php
if ( ! $addMetaColumns && ! isset($assoc['id'])) {
```

Reply:
Does it make sense to turn this into a single `IF` with an `OR` instead? Like:

```php
if ( ! ($assoc['isOwningSide'] && $assoc['type'] & ClassMetadata::TO_ONE)
    || ( ! $addMetaColumns && ! isset($assoc['id']))) {
```

I mean, both are fine... just wondering 😜

Reply:
@lcobucci thanks for your review, make sense to turn in the same if statement